core/kits/assets/css/panel.scss

```diff
@@ -10,7 +12,7 @@
 .elementor-editor-kit {
```

Comment:
It's highly recommended to move the whole "kit" style rules to a separated file

Reply:
?